pkgs/tools/security/vault/default.nix

```diff
@@ -7,27 +7,32 @@ let
     rev = "e2f59b64be1fa5430fa05c91b6274284de4ea77c";
     sha256 = "10m75rp3hy71wlmnd88grmpjhqy0pwb9m8wm19l0f463xla54frd";
   };
-in buildGoPackage rec {
+in stdenv.mkDerivation rec {
```

Comment:
This can be limited by `subPackages`. Looking into `build.sh` I suspect that `subPackages = [ "." ];` should produce the same bin file as `Makefile`.

I don't try to convince you to use `buildGoPackage`, just saying there are options to make it easier.